### DISCUSSION TITLE
Vector run on arm system with error

### DISCUSSION THREAD

**Discussion by FengZh61** [vectordotdev/vector#16645] [Q&A] [answered]
uname -a 
Linux localhost.localdomain 4.19.90-2201.4.0.0135.up1.uel20.aarch64 #1 SMP Mon Feb 21 18:36:06 UTC 2022 aarch64 aarch64 aarch64 GNU/Linux

./vector 
<jemalloc>: Unsupported system page size
<jemalloc>: Unsupported system page size
memory allocation of 5 bytes failed
Aborted (core dumped)

vector vesion 

0.27.0 gnu

**Comment by jszwedko** [comment, marked as answer]
This looks similar to https://github.com/vectordotdev/vector/issues/4392. Do you happen to be running on system with 64 KiB pages? Which OS are you running? You can see, on that issue, how you can rebuild Vector with a different page size if needed.

**Comment by ozgurkalan** [comment]
Hi, I am experiencing a similar error:

*pi and os:*
``raspbian - Raspberry Pi 5
Debian GNU/Linux 12 (bookworm)
``

*container:*
``` image: timberio/vector:0.28.1-alpine ```

*error:*
```
<jemalloc>: Unsupported system page size
memory allocation of 5 bytes failed
```
